src/cljs/rems/flash_message.cljs

```diff
@@ -1,9 +1,9 @@
 (ns rems.flash-message
   (:require [clojure.test :refer [deftest is]]
-            [reagent.core :as reagent]
+            [clojure.walk]
```

Comment:
What is `clojure.walk` required for?

Reply:
It has been used but not required previously in this same file:

https://github.com/CSCfi/rems/blob/58520dfc83ac7e1e7f09b09ad3e415cd6898ee87/src/cljs/rems/flash_message.cljs#L127-L132